hack/openshift/fetch-tektoncd-catalog-tasks.sh

```diff
@@ -34,7 +34,7 @@ USAGE:
     $SCRIPT_NAME DEST_DIR
 
 Example:
-  $SCRIPT_NAME cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_external
+  $SCRIPT_NAME cmd/openshift/operator/kodata/tekton-addon/addons/02-tasks/source_external
```

Comment:
we no more have this directory 02-tasks so can you use 
`$SCRIPT_NAME cmd/openshift/operator/kodata/tekton-addon/addons/07-ecosystem`